packages/core/components/FileMetadataSearchBar/index.tsx

```diff
@@ -0,0 +1,157 @@
+import {
+    DatePicker,
+    Dropdown,
+    Icon,
+    IconButton,
+    IDropdownOption,
+    SearchBox,
+} from "@fluentui/react";
+import * as React from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import { AnnotationName, TOP_LEVEL_FILE_ANNOTATIONS } from "../../constants";
+import { AnnotationType } from "../../entity/AnnotationFormatter";
+import FileFilter from "../../entity/FileFilter";
+import { selection } from "../../state";
+import { getFileAttributeFilter } from "../../state/selection/selectors";
+
+const styles = require("./FileMetadataSearchBar.module.css");
+
+const FILE_ATTRIBUTE_OPTIONS = TOP_LEVEL_FILE_ANNOTATIONS.filter(
+    (a) => a.name !== AnnotationName.FILE_SIZE
+).map((a) => ({
+    key: a.name,
+    text: a.displayName,
+}));
+const FILE_NAME_OPTION = FILE_ATTRIBUTE_OPTIONS.find(
+    (o) => o.key === AnnotationName.FILE_NAME
+) as IDropdownOption;
+// Color chosen from App.module.css
+const PURPLE_ICON_STYLE = { icon: { color: "#827aa3" } };
+export const DATE_RANGE_SEPARATOR = "-to-"; // Not arbitrary, defined per contract with FES
+
+// Because the datestring comes in as an ISO formatted date like 01-02-21
+// creating a new date from that would result in a date displayed as the
+// day before due to the UTC offset, to account for this we can add in the offset
+// ahead of time.
+function extractDateFromDateString(dateString?: string): Date | undefined {
+    if (!dateString) {
+        return undefined;
+    }
+    const date = new Date(dateString);
+    date.setMinutes(date.getTimezoneOffset());
+    return date;
+}
+
+/**
+ * This component renders a dynamic search bar for querying file records by
+ * basic file attributes that are otherwise not queryable through usual means like
+ * in the <AnnotationSidebar /> for example.
+ */
+export default function FileMetadataSearchBar() {
```

Comment:
When I use the search bar, copy the FMS Explorer URL, refresh the app and paste the URL back into the app, there isn't much visual indication of what filter is applied. And equally important, unless I know how the URL string works, I don't have a way to clear the filter.  IMO, this is an important piece to put into place before the feature is ready. I see two options:
1. Populate the search bar with the attribute filter applied, if any.
2. Figure out how to shoehorn attribute filters into the "filter display bar" 

Option 1 seems more straightforward (both as a developer and user of the application) to me?